portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,35 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body, html {
+  height: 100%;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.bg {
+    background-image: url("/images/mountain.JPG");
+    filter: blur(4px);
+    -webkit-filter: blur(4px);
+    height: 100%;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
 }
+
+#home_content {
+    position: absolute;
+    top: 50%;
+    left: 50%;
```

Comment:
Use "inset" instead

https://developer.mozilla.org/en-US/docs/Web/CSS/inset

https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties

Reply:
I tried inset: 50% 50% and it messed everything up, is that the correct syntax?

Reply:
I tried inset but it moved my box to the bottom left of the page.  I used the syntax inset: 50% 50% is this correct?

Reply:
It should be insert: 50% 0% 0% 50% I believe.